cypress/e2e/app.cy.ts

```diff
@@ -399,29 +399,29 @@ describe('/mock', () => {
       }
     });
 
-    it('visualize 2D complex signal as NxImage', () => {
```

Comment:
I've swapped the two mock complex `NXdata` groups to have the spectrum first and the image second, for consistency with the other mock `NXdata` groups, and I've renamed the `complex` `NXdata` group to `complex_image` for clarity.

This is why all the snapshots have changed below and why the diff of `app.cy.ts` looks weird here (since I've swapped the specs as well).